byway/src/main/webapp/destinations.js

```diff
@@ -155,18 +155,18 @@ function updateLocations(locationData){
 /** 
 * adds Users Input Destinations to DOM with image and address
 */
-function addLocationToDom(results,container){
+function addLocationToDom(place,container){
```

Comment:
It seems like container is not used in the calling function, so instead of passing it as a param just declare it inside this function directly.

Reply:
I do use it in line 150 because I clear the container before adding all of the locations the user has input